### ISSUE TITLE
pyjson5 should support negative numbers

### ISSUE THREAD

**Issue by aviafelix** [dpranke/pyjson5#11]
Current version of **json5** module has no support for negative numbers (moreover for numbers with minus sign including zero). It works fine for positive numbers and zero without minus:
```python
>>> import json5
>>> json5.version.VERSION
'0.2.4'
>>> json5.loads('5')
5
>>> json5.loads('5.5')
5.5
```
but breaks when there is a minus sign:
```python
>>> json5.loads('-5')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/aviafelix/Downloads/nu_e_to_eminus_wplus/pyjson5/json5/lib.py", line 29, in loads
    raise Exception(err)
Exception: :1:1 expecting '0X'
>>> json5.loads('-0.0')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/aviafelix/Downloads/nu_e_to_eminus_wplus/pyjson5/json5/lib.py", line 29, in loads
    raise Exception(err)
Exception: :1:1 expecting '0X'
```

**Comment by dpranke** [comment]
Yup, that's a bug :).

**Comment by dpranke** [comment]
This should be fixed in the latest commit, 3cb64842, which I've tagged with `v0.3`. I will do another release to PyPI once I've done a bit more testing and fixed a couple more issues, hopefully in a couple days.
